src/main/webapp/data-visualization.js

```diff
@@ -296,3 +302,47 @@ function setStyle(isCountyQuery) {
     amChartsDiv.style.display = 'block';
   }
 }
+
+function initDataTable(isCountyQuery) {
```

Comment:
This is a simple and good start. For next steps, you could also consider using google table visualization (https://developers.google.com/chart/interactive/docs/gallery/table) to display the table. It allows you sorting by any column automatically.